yield_plugin/src/yield_plugin.cpp

```diff
@@ -232,28 +232,30 @@ namespace yield_plugin
     if (req->initial_trajectory_plan.trajectory_points.size() < 2){
       throw std::invalid_argument("Empty Trajectory received by Yield");
     }
+    std::chrono::system_clock::time_point start_time = std::chrono::system_clock::now();  // Start timing the execution time for planning so it can be logged
```

Comment:
Is there a reason we are accessing the system clock directly instead of going through ROS? From my understanding, the ROS API provides access to system time.